.github/workflows/easy-cp.yml

```diff
@@ -58,8 +52,8 @@ jobs:
         if: ${{ steps.attempt-cp.conclusion == 'success' }}
         working-directory: ./flutter
         run: |
-          git push -u origin cp-${CHANNEL}-${COMMIT_SHA}
-          gh pr create --title "[CP-${CHANNEL}]${PR_TITLE}" --body-file ../PULL_REQUEST_CP_TEMPLATE.md --base ${RELEASE_BRANCH} --label "cp: review" --repo flutter/flutter
+          git push https://${GITHUB_TOKEN}@github.com/flutteractionsbot/flutter cp-${CHANNEL}-${COMMIT_SHA}
```

Comment:
Does `${GITHUB_TOKEN}@github.com` leak the secret in the logs?

Reply:
nope. it outputs ***@github.com where the raw string is replaced with asterisks. [example workflow](https://github.com/XilaiZhang/miscellaneous-side-project/actions/runs/7548409479/job/20550397014)